src/js/routes.js

```diff
@@ -37,9 +38,12 @@ export default (
              onEnter={redirectIfFlagIsDisabled.bind(null, flagChecks.contactUsIsEnabled())} />
       <Route path="/blog" component={Blog} />
       <Route path="/not-found" component={NotFound} />
+      <Route path="about-us"
+             component={AboutUsPage} onEnter={redirectIfFlagIsDisabled.bind(null, flagChecks.aboutUsIsEnabled())} />
     </Route>
+    <Route path="/not-found" component={NotFound} />
+
```

Comment:
Adding this flag is awesome! ❤️ 
Have the `ABOUT_US_ENABLED` flag been added to Heroku yet?